src/main/java/org/jenkinsci/plugins/prometheus/context/Context.java

```diff
@@ -0,0 +1,16 @@
+package org.jenkinsci.plugins.prometheus.context;
+
+import com.google.inject.AbstractModule;
```

Comment:
Explicit use of Guice in Jenkins plugins is almost always a bad idea. Whatever the issue was here (reproducible?), this was probably not the right way to solve it.

Reply:
this pr is quite old, this class no longer exists in current versions